grails-app/views/wps/jobComplete.gsp

```diff
@@ -1,6 +1,8 @@
 Hi,
 
-Your IMOS download is ready. You can download it by clicking here: ${jobReportUrl}
+Your IMOS subset has been created. You can download it by clicking here: ${jobReportUrl}
+
+Note: The link will only be available for the next 24 hours.
```

Comment:
Not sure that I explained myself properly before, but the "24" should be coming from `Config.groovy`.  This will then make it easy for chef to overwrite (you don't have to worry about *that* for now, but it'd be nice if this was configurable to begin with).

It's called `wps.resourceExpirationTimeout` in geoserver, not sure if that's descriptive enough.